generic/ltp.py

```diff
@@ -58,29 +60,26 @@ def test(self):
         args = self.params.get('args', default='')
         logfile = os.path.join(self.logdir, 'ltp.log')
         failcmdfile = os.path.join(self.logdir, 'failcmdfile')
+        failed_tests = False
```

Comment:
Can we replace `values` list as `failed_tests` list and avoid using `failed_tests` bool?